lesson08/assignment/bakers-home/src/components/CommunityView.vue

```diff
@@ -0,0 +1,68 @@
+
+<template>
+  <div id="app">
+    <main class="main">
+      <!-- image carousel with three images and controls -->
+      <h3>Community Bread Gallery</h3>
+      <div id="carouselControls" class="carousel slide" data-ride="carousel">
+        <div class="carousel-inner" role="listbox">
+          <!-- picture component, looping through array of images -->
+          <Picture 
+            v-for="(pic, index) in pics" 
+            :key="pic.id" 
```

Comment:
I don't think that pic.id exists.  The key for this for loop can use :key="index" instead.